client/app/cdn/dedicated/domain/update/translations/Messages_fr_FR.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<translations>
+   <translation id="cdn_dedicated_domain_bypass_confirm">Souhaitez-vous contourner temporairement le réseau CDN pour le domaine <b>{0}</b> ?</translation>
```

Comment:
`strong` instead of `b` ?

Reply:
Yes, I copy pasted it and as I reset the qtlid, I can also change this :-)